### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by AarshiyaGuneja**
Signed-off-by: AarshiyaGuneja <aarshiyaguneja@gmail.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+## Making A Change
+
+*Before making any significant changes, please [open an issue](https://github.com/jaegertracing/cpp-client/issues).*
+Discussing your proposed changes ahead of time will make the contribution process smooth for everyone.
+
+Once we've discussed your changes and you've got your code ready, make sure
+that tests are passing and open your pull request. 
+
+* Includes tests for new functionality.
```
Comment (yurishkuro) [inline CONTRIBUTING.md]: this bullet list is missing some intro sentence

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+By contributing to this project you agree to license your contribution under the terms
+of the [Apache License](LICENSE), and you agree to the [Developer Certificate of
+Origin](https://developercertificate.org/) (DCO). This document was created
+by the Linux Kernel community and is a simple statement that you, as a
+contributor, have the legal right to make the contribution. See the [DCO](DCO)
+file for details.
+
+/*
+ * Copyright (c) 2017, The Jaeger Authors
```
Comment (yurishkuro) [inline CONTRIBUTING.md]: 2018

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+We'd love your help!
+
+Jaeger is [Apache 2.0 licensed](LICENSE) and accepts contributions via GitHub
+pull requests. This document outlines some of the conventions on development
+workflow, commit message formatting, contact points and other resources to make
+it easier to get your contribution accepted.
+
+We gratefully welcome improvements to documentation as well as to code.
+
+## Making A Change
```
Comment (yurishkuro) [inline CONTRIBUTING.md]: normally we include instructions in this file on how to get started, i.e. which setup/dependencies are required, how to run tests, etc. However, we could add this later, after this PR is merged. -- /cc @isaachier

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
+# How to Contribute to Jaeger
```
Comment (yurishkuro) [inline CONTRIBUTING.md]: please add a section to README referencing this file

Comment (AarshiyaGuneja) [reply CONTRIBUTING.md]: Yes, Sure!

File: README.md [hunk 5]
```diff
@@ -27,3 +31,7 @@ https://issues.apache.org/jira/browse/THRIFT-4484
 [cov]: https://codecov.io/gh/jaegertracing/cpp-client
 [ot-img]: https://img.shields.io/badge/OpenTracing--1.0-enabled-blue.svg
 [ot-url]: http://opentracing.io
+
+## License
+  
+[Apache 2.0 License](./LICENSE).
```
Comment (yurishkuro) [inline README.md]: please move above the shortcuts

**Comment by AarshiyaGuneja** [comment]
Thanks! @yurishkuro

**Comment by yurishkuro** [comment]
Thanks, @AarshiyaGuneja

**Pull request merged**